mkdocs/docs/getting-started/identity/identity-types.md

```diff
@@ -119,7 +120,7 @@ The last two limitations can be overcome by adding BJJ keys to Claims Tree and p
 | Keys                               | BJJ keys                    | Ethereum Account (SC or EOA) | Ethereum Account (SC or EOA) + BJJ keys                    |
 | Authentication Method (off-chain)  | JWZ with ZKP using BJJ keys | JWS with Ethereum Signature  | JWS with Ethereum Signature or JWZ with ZKP using BJJ keys |
 | Authentication Method (on-chain)   | ZKP using BJJ keys          | Ethereum Account             | Ethereum Account or ZKP using BJJ keys                     |
-| State Transition Method            | ZKP using BJJ keys          | Ethereum Account             | Ethereum Account or ZKP using BJJ keys                     |
+| State Transition Method            | ZKP using BJJ keys          | Ethereum Account             | Only Ethereum Account                                      |
```

Comment:
Roll back this change.

Reply:
Ok
Let's add a warning like this?
\* note that neither State contract nor IdentityLib restrict users from doing state transition I`dentityLib.transitState()` followed by state transition via the `State.transitState()` contract directly with BJJ keys auth. It may cause inconsistent state data in between `State` and Onchain Identity smart contracts and it is responsibility of on-chain identity owner to avoid such situations.